<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VSystemParameter</class>
 <widget class="QWidget" name="VSystemParameter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tbSystem">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>861</width>
     <height>641</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Index</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Value</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Note</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="lstVersion">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>50</y>
     <width>861</width>
     <height>641</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblSysInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>System Information:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblVerInfo">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>10</y>
     <width>861</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Version Information:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
